resources/lib/NineAnimeBrowser.py

```diff
@@ -59,14 +61,21 @@ def _handle_paging(self, results, base_url, page):
 
         next_page = page + 1
         name = "Next Page (%d/%d)" % (next_page, total_pages)
-        return [utils.allocate_item(name, base_url % next_page, True, None)]
+        return [utils.allocate_item(name, base_url % next_page,'', True, None)]
+
+    #def _handle_plot(self, res):
```

Comment:
change it to "_extract_anime_plot(self, url)"
and uncomment, it's not used but might be in the future ;)